packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -231,7 +231,7 @@ export default defineComponent({
 					position: 'top',
 					available: this.settingsStore.isTemplatesEnabled,
 					link: {
-						href: WEBSITE_TEMPLATES_URL,
+						href: `${WEBSITE_TEMPLATES_URLS.BASE_URL}?${WEBSITE_TEMPLATES_URLS.UTM_QUERY}`,
```

Comment:
This is a bit hard to read. Could we do something like `getTemplateRepositoryUrl()` that builds the URL? The category could be a parameter or it's own function `getTemplateRepositoryUrl(AI_CATEGORY_ID)` / `getTemplateRepositoryAiCategoryUrl()`. We could then use the `URL` object to build the URL instead of string concat

Reply:
Agree